src/services/metrics.ts

```diff
@@ -0,0 +1,16 @@
+import { config } from "../services/config"
+import { MetricsService } from '@aerogear/core';
+
+
+export var metrics: MetricsService;
+
+// Create metrics
+export function initMetrics() {
+  // TODO we need shared instance on SDK level not app level!
+  // TODO we need this to not crash if config is missing
+  let metricsConfig = config.getConfig("metrics");
+  metrics = new MetricsService(metricsConfig);
```

Comment:
This is just an idea for future PRs, instead of passing the config manually, we could do a aerogear.init() that would read the config file and store it globally, then user would only have to do `new MetricsService()` and it internally would fetch the configuration. This way also we make the parser transparent to the user and we do not worry about `getConfig` or helper methods. I think in the end, the user should not worry about the parser at all, it is more like a indoor class for the services.

Reply:
This has many negative implications that we want to avoid. . We actually moving from that pattern on Android where many cloud apps needs to be created using